include/SFML/System/Time.hpp

```diff
@@ -86,9 +86,9 @@ class Time
 
 private:
 
-    friend constexpr Time seconds(float);
-    friend constexpr Time milliseconds(Int32);
-    friend constexpr Time microseconds(Int64);
+    friend constexpr Time seconds(float) noexcept;
+    friend constexpr Time milliseconds(Int32) noexcept;
+    friend constexpr Time microseconds(Int64) noexcept;
```

Comment:
I believe you're the first person to start (rightfully) decorating functions as `noexcept` where appropriate. This PR would be the first to introduce such changes. @eXpl0it3r @Bromeon @vittorioromeo is this a pattern we should start applying to everything which cannot throw?

Reply:
see main thread